check-vulnerabilities/action.yml

```diff
@@ -325,6 +319,12 @@ runs:
           python -m pip install ."${extra_targets[@]}"
         fi
 
+    - name: "Install action requirements"
+      shell: bash
+      run: |
+        ${{ env.ACTIVATE_VENV }}
+        pip install -r ${{ github.action_path }}/requirements.txt

```

Comment:
I know this was already, but it would be nice to add here:

```suggestion
        python -m pip install -r ${{ github.action_path }}/requirements.txt

```